modules/core/src/main/scala/io/chrisdavenport/cats/time/instances/zoneid.scala

```diff
@@ -5,9 +5,11 @@ import java.time.ZoneId
 
 trait zoneid {
   implicit final val zoneidInstances = 
-    new Show[ZoneId]{
+    new Eq[ZoneId] with Hash[ZoneId] with Show[ZoneId] {
```

Comment:
These can't be mixed in and keep binary compatibility. Since Eq/Hash don't share any types with Show this is fine and we can just place them in another implicit in the trait.

Reply:
Moved those to a separate val - but I'm still getting `ReversedMissingMethodProblem` locally.

Reply:
Reverted since binary compatibility is unobtainable (see #50)

Reply:
... "unobtainable" was too strong a word, apparently. All I had to do was to sort the traits so that the existing one (`Show`) came first. This proved to be enough to keep the bytecode compatible. Go figure.